packages/v4/patternfly-docs/content/accessibility/badge/badge.md

```diff
@@ -3,30 +3,61 @@ id: Badge
+To implement an accessible PatternFly **badge**:
 
-No props need to be added or modified for badge accessibility.
+- Ensure the badge itself is not interactive in any way. 
+- Ensure that the surrounding context can convey the purpose of the badge to users via screen reader or similar assistive technologies.
+- Ensure badge information is not conveyed by only color, such as a read or unread badge.
 
-**As a caution**, badges can be styled using a variety of colors. Do not rely on color alone to communicate information 
-because it causes barriers to access for many readers. For example, colorblind and low vision users may not be able 
-to perceive the color differences, and screen readers do not announce colors to non-sighted readers.
+## Testing
 
-Since the badge doesn't get an accessible name and isn't focusable, each badge must have adequate contextual information 
-provided in the surrounding UI to convey the same information the color alone is conveying. A developer could consider
-using a label with an icon in it to supplement the color.
+ At a minimum, a badge should meet the following criteria:
 
-Note how difficult it is to determine that there are unread or actionable items associated with either of the badges
-in Figure 1 with its lack of visual cues. Compare Figure 1 to Figure 2 where there is an icon
-used to indicate that the second badge is associated with actionable or unread content.
+<List isPlain>
+  <ListItem>
+    <Checkbox id="badge-a11y-checkbox-1" label="Users should not be able to interact with or navigate to the badge with the keyboard, as it is not an interactive element." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="badge-a11y-checkbox-2" label="Users navigating via screen reader or similar assistive technology should understand the purpose of the badge by its surrounding context." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="badge-a11y-checkbox-3" label="Information about a badge is not conveyed by only color."/>
+  </ListItem>
+</List>
+
+## React customization
+
+A badge does not have any further React props for accessibility.
+
+## HTML/CSS customization
+
+A badge does not have any further HTML/CSS attributes or classes for accessibility.
+
+## Additional considerations
+
+Consumers must ensure they take any additional considerations when customizing a badge, using it in a way not described or recommended by PatternFly, or in various other specific use cases not outlined elsewhere on this page.
+
+### Color
+
+Generally you should avoid using only color to convey information about a badge. One possible exception to this rule may be if a badge is only ever rendered in a specific scenario, such as only when there are unread notifications, and will otherwise not be rendered for any other purpose. Caution should still be used when using color in this way, though.
```

Comment:
So I'm not completely sure what you mean here. Are you trying to say in a scenario in which the color of the badge doesn't genuinely matter for understanding?

Reply:
More or less, yeah. This could probably just be condensed to something like, "Generally you should avoid using only color to convey information about different types of badges, such as between read and unread notifications."

Do you have a suggestion to update this verbiage?

Reply:
Although color is generally an accessibility issue, I think the high contrast of the two types of badge plus accessible labeling addresses a11y issues. The understandability of which badge means what seems to be more of a _usability_ issue, so perhaps that part could be moved over to the design tab?

Reply:
That's true... I could see the info technically working in either place. The Badge now supporting a `screenReaderText` prop in React and examples showing that visually hidden text in action definitely help, and I can update both the HTMl and React tables to mention the appropriate attribute/class/prop. 

I could also update the testing item about color to add a description mentioning how icons and visually hidden text can/should be used. @jessiehuff @srambach wdyt?

Reply:
Also, I do think that it could make sense in both sections, but I've always found it important to remind users not to use colors to communicate information in different use cases. I think a lot of designers and developers forget about how it affects a11y so I think it's important that they're not forgetting to add contextual information, either in the form of surrounding UI or additional hidden text with labels or screen reader text.

Reply:
@jessiehuff I'd be for omitting the second bit and just keeping the "Generally you should avoid using only color to convey information about different types of badges, such as between read and unread notifications" portion. Do you think the paragraph that follows ("If a badge will always be rendered whether it's unread or read...") is fine as-is, or could more be tweaked in this section?

Reply:
Yeah, I think that makes sense. I'm open to suggestions though if you or others think differently. :)

Reply:
@jessiehuff let me know what you think of the latest updates! I added some verbiage + a couple of quick examples re: context to the first section, and updated the Colors section to begin with the verbiage you commented with here